docs/UserGuide.md

```diff
@@ -54,35 +55,92 @@ not be so obvious!
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`c-list`** : Lists all employees.
+   * **`list`** : Lists all contacts.
 
-   * **`c-add`**`n/John Doe p/98765432 e/81234567 t/parttime` : Adds an employee
+   * **`c-add`**`n/John Doe p/98765432 e/81234567 t/part time` : Adds an employee's contact
     named `John Doe` to tCheck.
 
-   * **`c-delete`**`3` : Deletes the 3rd employee shown in the current list.
+   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
 
-   * **`c-clear`** : Deletes all employees.
+   * **`clear`** : Deletes all contacts.
 
    * **`exit`** : Exits the app.
 
 1. Refer to the [Features](#features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
+## 3. About <a name="about"></a>
 
+### 3.1 Structure of this document
+
+We have structured this User Guide in a way so that you can find what you need easily and quickly. In the next section,
+[Section 3.2 - Reading this document](#reading-this-document), you can find sevaral useful tips on how to read this guide. The following section,
+[Section 4 - Features](#features), documents the four main features in **tCheck**, namely:
+
+   * Sales Tracking
+
+   * Ingredients Tracking
+
+   * Employee Management
+
+   * Other general features
+
+### 3.2 Reading this document <a name="reading-this-document"></a>
+
+This section introduces you to some key technical terms, symbols and syntax that are used throughout this guide.
+You may want to familarize yourself with them before moving on to the next section.
+
+#### 3.2.1 Terminology related to the GUI
+
+Figure 2 shows the GUI of **tCheck**, annotated with descriptions for all components of the GUI.
+
+<img src="images/tCheckInfographic.png" />
+Figure 2 - Annotated GUI of tCheck
+
+#### 3.2.4 Prefix process and usage
+
+The table below explains some important features of how prefix(es) are processed in tCheck to help you understand and use prefix(es) in commands.
+
+Situation | What will happen
+-------|------------------------------
+Duplicate (2 or more) valid prefixes detected | Only the last prefix and its following parameter are accepted and processed as part of the command.
+Invalid prefix entered | tCheck will not recognize this "prefix" and thus it will be processed as part of the parameter for the nearest previous valid prefix, which may result in error for the parameter.
+Unexpected prefix(es) entered for commands which do not take in any parameter | tCheck will not accept or process these prefix(es) because all words entered after the command words will be ignored. 
+
+**Example 1:**
+`i-set i/Milk i/Boba m/90`
+**How is the command processed:**
```

Comment:
Should not be a question. A better phrasing is "How the command is processed"

Reply:
I agree, will change.